### ISSUE TITLE
Indexing Guidelines

### ISSUE THREAD

**Issue by nelong** [active-calculus/active-calculus-vector#42]
We need to establish some guidelines for when to use index tags and examples of how to use the index tags in different ways to connect users to relevant info.

**Comment by davidfarmer** [comment]
The PreTeXt documentation has some good advice, and several PreTeXt
authors took a short seminar on indexing.  (The documentation is
based on the seminar.)

https://pretextbook.org/doc/guide/html/topic-index.html

The AC authorities may have some AC-specific suggestions, if that is what 
was meant.

**Comment by mitchkeller** [comment]
Yeah, @davidfarmer, this was created as a result of a meeting @nelong and I just had. It's very much a "we need to decide on what we're doing for this particular book" task.
